services/shhext/service.go

```diff
@@ -67,6 +79,36 @@ func (s *Service) Protocols() []p2p.Protocol {
 	return []p2p.Protocol{}
 }
 
+func (s *Service) InitProtocol(address string, password string) error {
+	if s.pfsEnabled {
+		if err := os.MkdirAll(filepath.Clean(s.dataDir), os.ModePerm); err != nil {
+			return err
+		}
+		persistence, err := chat.NewSQLLitePersistence(filepath.Join(s.dataDir, fmt.Sprintf("%x.db", address)), password)
+		if err != nil {
+			return err
+		}
+		s.protocol = chat.NewProtocolService(chat.NewEncryptionService(persistence, s.installationID))
+	}
+	return nil
+}
+
+func (s *Service) ProcessPublicBundle(myIdentityKey *ecdsa.PrivateKey, bundle *chat.Bundle) error {
+	if s.protocol == nil {
+		return errors.New("Procotol is not initialized")
```

Comment:
Error could be extracted to a variable. Error message should start with lowercase.